pharus/interface.py

```diff
@@ -504,7 +557,12 @@ def _set_datajoint_config(jwt_payload: dict) -> dj.connection.Connection:
         :return: DataJoint connection object.
         :rtype: :class:`~datajoint.connection.Connection`
         """
-        dj.config["database.host"] = jwt_payload["databaseAddress"]
-        dj.config["database.user"] = jwt_payload["username"]
-        dj.config["database.password"] = jwt_payload["password"]
+        if not host_name:
+            dj.config["database.host"] = jwt_payload["databaseAddress"]
+            dj.config["database.user"] = jwt_payload["username"]
+            dj.config["database.password"] = jwt_payload["password"]
+        else:  # oidc jwt login
+            dj.config["database.host"] = host_name
+            dj.config["database.user"] = jwt_payload["sub"]
```

Comment:
`sub` is not a uniform field for subject in OIDC. We likely need to specify a new env var for which key specifies the user. Perhaps: `PHARUS_OIDC_SUBJECT_KEY`.